main.tf

```diff
@@ -74,3 +86,10 @@ module "ci_alerts_for_production" {
   topic_name              = "ci_alerts_for_production"
   subscription_account_no = local.accounts.production.id
 }
+
+module "ci_alerts_for_central_audit_development" {
+  source = "./modules/alerting_sns_topics"
+
+  topic_name              = "ci_alerts_for_central_audit_development"
+  subscription_account_no = local.accounts.production.id
```

Comment:
Is using a production topic here deliberate?

Reply:
Oooh good spot - I'll have a look! Thank you!